homeassistant/components/binary_sensor/deconz.py

```diff
@@ -113,3 +115,21 @@ def device_state_attributes(self):
         if self._sensor.type in PRESENCE and self._sensor.dark is not None:
             attr[ATTR_DARK] = self._sensor.dark
         return attr
+
+    @property
+    def device(self):
+        """Description for device registry."""
+        if (self._sensor.uniqueid is not None and
+                self._sensor.uniqueid.count(':') == 7):
+            mac = self._sensor.uniqueid.split('-', 1)[0]
+        else:
+            return None
+        dev = {
+            'connection': [[CONNECTION_ZIGBEE, mac]],
+            'identifiers': [[IDENTIFIER_MAC, mac]],
```

Comment:
There should not be an identifier MAC. There is already a connection one. 

Is there a serial number for this sensor? In that case you could add `[DECONZ_DOMAIN, serial]`

Also MAC should never be used as just a standalone type, as there is Network MAC, Zigbee MAC etc.

Reply:
Good point with DECONZ_DOMAIN since it should only map inside of deconz context 👍 

Previously you asked if the sensor had a mac, I will go back to using serial :)

Should I just call them NETWORK_MAC and ZIGBEE_MAC?

Reply:
yes